### ISSUE TITLE
Feature: Make the application's name more readable in the sealos data directory at /var/lib/sealos/data/default/applications

### ISSUE THREAD

**Issue by tcfly** [labring/sealos#4484]
### What is the problem this feature will solve?

![image](https://github.com/labring/sealos/assets/49434080/7c3950cf-291b-4aa9-9bea-6cbd299ceba9)

default-template,default-terminal...maybe better

**Comment by bxy4543** [comment]
The user does not seem to need to be aware of the directory, and the same situation exists for the name

**Comment by tcfly** [comment]
I want to uninstall the terminal application from my self-hosted sealos, just need to find out how to do it.

**Comment by bxy4543** [comment]
Yes, you can also query the corresponding directory by executing `sealos ps`

**Comment by tcfly** [comment]
Thx

**Comment by willzhang** [comment]
Same issue: https://github.com/labring/sealos/issues/2433

The fastest way to accurately find the image path currently

```
sealos ps -f ancestor=cni-plugins --notruncate
```
